src/TemplateLibrary/index.js

```diff
@@ -95,12 +96,20 @@ const TemplateLibraryComponent = (props) => {
 
   const filterTemplates = (templates) => {
     let filteredTemplates = templates;
+
     if (query.length) {
       const regex = new RegExp(query, 'i');
       filteredTemplates = _.filter(filteredTemplates, t => (
         t.name.match(regex) || t.uri.match(regex)
       ));
     }
+
+    if (props.semver && props.semver.length) {
+      filteredTemplates = filteredTemplates.filter(t => (
+        satisfies(t.version, props.semver)
```

Comment:
This is filtering based on the version of the template - what we would like it to filter on the ciceroVersion used by the template. I think changing this to `t.ciceroVersion` should do the trick. Please also rename the prop to be `ciceroSemanticVersionFilter` to be more explicit.